skyportal/app_server.py

```diff
@@ -457,6 +459,8 @@ def log_request(self, handler):
         ObservationPlanFieldsHandler,
     ),
     (r"/api/objs(/[0-9A-Za-z-_\.\+]+)", ObjHandler),
+    (r"/api/objtagoption(/.*)?", ObjTagOptionHandler),
```

Comment:
for routes that expect numbers only as parameters, we should enforce that so you don't get ids that make no sense. Could just be  something like`(/[0-9]+)?`.